rfc021-platform-support-policy.md

```diff
@@ -48,14 +48,11 @@ Ubuntu Linux | 12.04, 14.04 | x86, x86_64 | deb | Ubuntu 12.04
 
 Tier 2 supported platforms are those on which Omnitruck will serve packages, but those packages may not have been built on that OS variant. Additionally, we may or may not do post-build verification on these platforms. A platform must have at least one Maintainer before being added to this tier.
 
-* SUSE Linux Enterprise Server 10, 11
+* SUSE Linux Enterprise Server 11, 12
 * Scientific Linux 5.x, 6.x and 7.x (i386 and x86-64)
-* Gentoo Linux (rolling release)
-* Arch Linux (rolling release)
 * Fedora (current non-EOL revisions)
-* OpenSUSE 13.1/13.2
+* OpenSUSE 13.1/13.2/42.1
```

Comment:
@tas50 has looked into it.  I'm pretty sure it fits the standards for this tier.  I think it reports something like "opensuseleap" right now, and that should probably be changed to "opensuse" which is an open issue.  I don't think that means we don't support it, because the packages we ship probably run on it find otherwise.